packages/models/src/lib/plugin-config-runner.ts

```diff
@@ -24,3 +24,16 @@ export const runnerConfigSchema = z.object(
 );
 
 export type RunnerConfig = z.infer<typeof runnerConfigSchema>;
+
+export const onProgressSchema = z
+  .function()
+  .args(z.unknown())
```

Comment:
Should the argument really be of type `unknown`? As user, this means I have no clue what I should pass in. I notice one test uses `'update'`, so is it a `string`? And why would I want to pass anything in? Genuine questions, there's no description.

Reply:
For `RunnerConfig` it could be `string` `StringBuffer` and it is the output of the onStdin event in most of the cases. 
If `RunnerFuntion` is used it can be anything basically as it is up to the plugin author